src/main/java/org/wise/portal/domain/peergrouping/logic/DifferentIdeasLogic.java

```diff
@@ -0,0 +1,27 @@
+package org.wise.portal.domain.peergrouping.logic;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class DifferentIdeasLogic {
+
+  private String componentId;
+  private String nodeId;
+
+  public DifferentIdeasLogic(String logic) {
+    Pattern pattern = Pattern.compile("\"(\\w+)\"");
```

Comment:
We might be able to change the regex to something like ```differentIdeas\("(\w+)",\s*"(\w+)"\)```

So the code might be something like this (I'm not sure if the escaping is correct).
```Pattern pattern = Pattern.compile("differentIdeas\\(\"(\\w+)\",\\s*\"(\\w+)\"\\)");```